package.json

```diff
@@ -43,29 +44,6 @@
   "engines": {
     "node": "^18.12 || ^20.9"
   },
-  "prettier": {
-    "arrowParens": "avoid",
-    "overrides": [
-      {
-        "files": [
-          "packages/ERTP/**/*.{js,ts}",
-          "packages/inter-protocol/**/*.{js,ts}",
-          "packages/store/**/*.{js,ts}",
-          "packages/vats/**/*.{js,ts}"
```

Comment:
Ah. I see that we're preserving the choice. Just moving where it is expressed. For this PR then, nevermind. But I'm still curious: should this be everything?